web/src/compose_validate.js

```diff
@@ -18,13 +18,18 @@ import * as people from "./people";
 import * as settings_config from "./settings_config";
 import * as settings_data from "./settings_data";
 import * as stream_data from "./stream_data";
+import * as ui from "./ui";
 import * as util from "./util";
 
 let user_acknowledged_wildcard = false;
 let wildcard_mention;
 
 export let wildcard_mention_large_stream_threshold = 15;
 
+function append_compose_banner_to_banner_list(new_row) {
+    ui.get_content_element($("#compose_banners")).append(new_row);
+}
```

Comment:
This function belongs in `compose_banner.ts`; I think that's blocked by `ui.js` (well, now scroll_util.js) not being TypeScript yet though, so maybe we can just move it once that's .